src/ui.rs

```diff
@@ -1275,7 +1283,10 @@ pub(crate) mod ui_context {
                 || is_key_pressed(KeyCode::LeftControl)
                 || is_key_pressed(KeyCode::RightControl)
             {
-                ui.key_down(megaui::KeyCode::Control, shift, ctrl);
+                ui.key_down(megaui::KeyCode::Control, shift, ctrl, cmd);
+            }
+            if ui.input.quit {
+                get_context().quit_requested = true;
```

Comment:
macroquad::ui is a module about drawing game UI, I am not quite sure if handling platform quit request belongs here?

Reply:
I am inclined to agree. Would this be better moved to something like begin_frame() in src/lib.rs, or could you make a suggestion where a platform quit request would be better handled?

Reply:
begin_frame looks like a right place to me. 

Justa side question, but is manual handling Cmd-Q a common practice on MacOs? Isnt there some sort of a system-level hotkeys for this kind of things?

Reply:
I only thought about that after I last commented. I can do some research; I bet I can find something. I will get back to this in a day or two with relevant changes.